### ISSUE TITLE
error: unknown option `allow-unrelated-histories'

### ISSUE THREAD

**Issue by yegor256** [yegor256/0pdd#73]
after #71 

```
error: unknown option `allow-unrelated-histories'
```

**Comment by yegor256** [comment]
@rultor release, tag is `0.16.6`

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/0pdd#73` was not in scope

**Comment by KrisRoofe** [comment]
refer to this stackoverflow post http://stackoverflow.com/questions/41356766/unknown-option-allow-unrelated-histories
